src/js/allcomponents.stories.js

```diff
@@ -74,280 +74,258 @@ const themes = {
   v1,
 };
 
-class Components extends Component {
-  state = {
-    baseSize: 24,
-    checkBox: true,
-    radioButton: 'RadioButton 1',
-    rangeSelector: [1, 2],
-    themeName: 'grommet',
-  };
+const Components = () => {
+  const [baseSize, setBaseSize] = useState(24);
+  const [checkBox, setCheckBox] = useState(true);
+  const [radioButton, setRadioButton] = useState('RadioButton 1');
+  const [rangeSelector, setRangeSelector] = useState([1, 2]);
+  const [themeName, setThemeName] = useState('grommet');
+  const [background, setBackground] = useState('default');
```

Comment:
background currently has a placeholder that says 'background' so it will indicate to the user what this Select container is about. 
Can we keep this refactored behavior with the 'background' placeholder rather than immediately assigning the `default` background.

Reply:
<img width="1398" alt="Screen Shot 2019-10-03 at 4 44 37 PM" src="https://user-images.githubusercontent.com/6320236/66169281-27479500-e5fd-11e9-9ee8-7286bed0b9bf.png">
This is the changes on the snapshot

Reply:
Ah, sure. I can see how that would be helpful.